lib/deaf_grandma.rb

```diff
@@ -14,18 +14,21 @@ def run!
     loop do
       user_input = get_user_input
       p speak(user_input)
```

Comment:
@sjreich `p` gives you the same value as `.inspect` but without `puts`'s newline.